### ISSUE TITLE
[Bug] Generic/MultipleStatementAlignment: does not end block at conditionals

### ISSUE THREAD

**Issue by jrfnl** [squizlabs/PHP_CodeSniffer#1937]
X-posting from https://github.com/squizlabs/PHP_CodeSniffer/issues/1925#issuecomment-372977308 as that issue is closed.

The fix in commit 860196ad898b8e5e9a7a79c28be470451af43961 introduced a new issue with the `Generic.Formatting.MultipleStatementAlignment` sniff.

Given this code:
```php
$abc = 'something';
if ( $something ) {
    // do something
} else {
   // do something else
}
$defghi = 'longer something;'
```

The sniff now appears to expects the assignment operators for the `$abc` and the `$defghi` variables to be aligned.

**Comment by gsherwood** [comment]
I couldn't replicate that error message with that code sample for some reason (not even in the unit tests), but I could with a shorter one:
```
$abc = 'something';
if ($foo) {
}
$defghi = 'longer something;'
```
I'll take a look at what's going on there.

**Comment by gsherwood** [comment]
Also, thanks for picking this up :)

**Comment by gsherwood** [comment]
That should be fixed now. Please let me know if not though, especially given that your code block didn't error for me, although the fix should still work for it.

**Comment by jrfnl** [comment]
@gsherwood Thanks for the quick response! I've just tested it on one of the two projects in which I saw this happening and with the fix included, things look to be working as expected again. :heart:

**Comment by jrfnl** [comment]
And just ran it against the second project I noticed new issues being reported in and can confirm that the new "issues" are gone and the all is "back to normal".

**Comment by gsherwood** [comment]
Thanks a lot for doing that testing.
